javascript/javascript3/week3/index.js

```diff
@@ -0,0 +1,165 @@
+const refs = {
+    input: document.querySelector('[name="website"]'),
+    makeScreenshotBtn: document.querySelector('[name="make-screenshot"]'),
+    saveScreenshotBtn: document.querySelector('[name="save-screenshot"]'),
+    libraryScreenshotsBtn: document.querySelector('[name="library-screenshots"]'),
+    libraryList: document.querySelector('.screenshots-list')
+};
+
+const BASE_URL_SCREENSHOT_API = 'https://website-screenshot6.p.rapidapi.com/screenshot/';
+const KEY_API_SCREENSHOT = 'default-application_7160449';
+
+const BASE_URL_CRUD_CRUD = ' https://crudcrud.com/api';
+const KEY_API_CRUD_CRUD = '5b127da1ec9b44eab72f4d5992846501';
+
+let screenshotUrl = '';
+    
+refs.makeScreenshotBtn.addEventListener('click', onMakeScreenshotBtnClick);
+refs.saveScreenshotBtn.addEventListener('click', onSaveScreenshotBtnClick);
+refs.libraryScreenshotsBtn.addEventListener('click', onLibraryScreenshotsBtnClick);
+refs.libraryList.addEventListener('click', onDeleteBtnClick);
+
+
+
+function onMakeScreenshotBtnClick() {
+    
+    refs.libraryList.innerHTML = "";
+    refs.libraryList.innerHTML = "Wait, it's might take some time";
+
+    fetchScreenshotApi(refs.input.value).then(data => {
+        createOneScreenshotMarkup(data.screenshotUrl);
+        screenshotUrl = data.screenshotUrl;
+        console.log(screenshotUrl);
+        if (screenshotUrl === undefined) {
```

Comment:
When checking whether a value is defined or undefined, you can simply use `if (!screenshotUrl)`.

This is because of the concept of Falsy values - see here: https://developer.mozilla.org/en-US/docs/Glossary/Falsy